packages/commonwealth/client/styles/pages/manage_community/manage_roles.scss

```diff
@@ -4,15 +4,14 @@
   display: flex;
   flex-direction: column;
   margin-bottom: 24px;
-  width: 248px;
 
   .roles-container {
     border: 2px solid $neutral-200;
     border-radius: $border-radius-corners;
     height: 196px;
     overflow-y: auto;
     padding: 0 4px;
-    width: 608px;
+    width: 100%;
```

Comment:
this will effect existing components, lets make sure we use different css value for width based on the feature flag, ex: `width: 100%;` when flag is enabled else the older `width: 608px;`